### ISSUE TITLE
Android input issue where bottom sheet moves upon keyboard opening

### ISSUE THREAD

**Issue by hannojg** [gorhom/react-native-bottom-sheet#68]
# Bug

Through these styles:
`src/components/BottomSheet/styles.ts`
```js
  container: {
    position: 'absolute',
    left: 0,
    right: 0,
    bottom: 0,
  }
```

it happens that when opening the Keyboard on android (e.g. having a text input in the bottom sheet's content) the content gets pushed up:
![BOTTOM](https://user-images.githubusercontent.com/16821682/99562698-b6ae3400-29c8-11eb-9e27-a6c28bb20c4e.gif)

This is the default behaviour on android. Interestingly, this doesn't happen when changing the styles from `bottom` -> `top`.
```js
  container: {
    position: 'absolute',
    left: 0,
    right: 0,
    top: 0,
  }
```

![TOP](https://user-images.githubusercontent.com/16821682/99562720-bf066f00-29c8-11eb-96dd-4a9583b9ac97.gif)

However, this breaks the BottomSheet, as all calculations are thought to be run from the bottom edge.
I am aware that it is possible to change on android the windowSoftInputMode, however, this is quite invasive and doesn't work with projects using react native navigation.

## Environment info

| Library                         | Version |
| ------------------------------- | ------- |
| @gorhom/bottom-sheet         | 1.4.1   |
| react-native                    | 0.63.3   |
| react-native-reanimated         |  2.0.0-alpha.8 |
| react-native-gesture-handler    | 1.8.0  |

## Reproducible sample code

Please tell me whether reproduction is necessary.

**Comment by gorhom** [comment]
thanks @hannojg for submitting this issue.

this already been resolved with #81, I have refactored the sheet styling to use `top` instead `bottom`. 

it will be released with v2 soon 👍

**Comment by danielmark0116** [comment]
To anybody who has issues with this (active keyboard moving the modal up), you can try to programmatically switch softInputMode to `adjustNothing` by writing a native module on a screen where you want to use the bottom sheet. Note that none keyboard events will be sent to the RN though

> I am aware that it is possible to change on android the windowSoftInputMode, however, this is quite invasive and doesn't work with projects using react native navigation.
